lib/paths-cache.js

```diff
@@ -223,13 +223,10 @@ export default class PathsCache extends EventEmitter {
     this._cancelled = false
     this.emit('rebuild-cache')
 
-    return execPromise('find --version')
-      .then(
-        // `find` is available
-        () => this._buildInitialCacheWithFind(),
-        // `find` is not available
-        () => this._buildInitialCache()
-      )
+    return this._buildInitialCacheWithFind().catch(
+      // `find` is not available
+      () => this._buildInitialCache()
+    ),
```

Comment:
I prefer that we keep `then` and insert this catch after it. On Windows, for example, I belive _buildInitialCacheWithFind will always fail

Reply:
Or maybe, move the try-catch inside `_buildInitialCacheWithFind`

Reply:
`find --version` always fails on Windows!
![image](https://user-images.githubusercontent.com/16418197/102104743-28c13f80-3df4-11eb-84b6-bd255981d22d.png)

Reply:
Maybe we should just don't call `execPromise('find --version')` on Windows!

Reply:
I changed it to check `process.platform === 'win32'` first